app/main.py

```diff
@@ -8,20 +8,19 @@ def __init__(self, name: str, age: int) -> None:
 
 
 def create_person_list(people_list: list) -> list:
-    person_instances = []
-    for person_dict in people_list:
-        person = Person(person_dict["name"], person_dict["age"])
-        person_instances.append(person)
+    person_instances = [Person(person_dict["name"], person_dict["age"])
+                        for person_dict in people_list]
```

Comment:
move the body inside list comprehension on separate lines as we do for function parameters, etc